cachi2/core/package_managers/generic/main.py

```diff
@@ -0,0 +1,81 @@
+import logging
+
+import yaml
+from pydantic import ValidationError
+
+from cachi2.core.errors import PackageManagerError, PackageRejected
+from cachi2.core.models.input import Request
+from cachi2.core.models.output import RequestOutput
+from cachi2.core.models.sbom import Component
+from cachi2.core.package_managers.generic.models import GenericLockfileV1
+from cachi2.core.rooted_path import RootedPath
+
+log = logging.getLogger(__name__)
+DEFAULT_LOCKFILE_NAME = "cachi2_generic.yaml"
```

Comment:
This will have to be renamed something else - it's part of the public API and we already know we're going to rebrand the project which means we'd have to rename the file as well -> BAD! Something like `generic_fetcher_lockfile.yaml` OR `generic_lockfile.yaml` might be good options, @brunoapimentel, @a-ovchinnikov  what do you think of the alternative names?

Reply:
Either what you suggested, or just `generic-artifacts.yaml`. In any case, we don't need to settle on a name right now, since this is still under the experimental flag.

Reply:
The development could move fast and so I'd like to at least get rid of the project reference ASAP otherwise we forget.

Reply:
Renamed to `generic_lockfile.yaml`